cmd/logs/cmd.go

```diff
@@ -5,16 +5,19 @@ import (
 	"os"
 
 	"numerous.com/cli/cmd/args"
+	"numerous.com/cli/cmd/output"
 	"numerous.com/cli/internal/app"
+	"numerous.com/cli/internal/dir"
 	"numerous.com/cli/internal/gql"
 
 	"github.com/spf13/cobra"
 )
 
 var LogsCmd = &cobra.Command{
-	Use:   "logs [app directory]",
-	Run:   run,
-	Short: "Display running application logs",
+	Use:     "logs [app directory]",
+	Run:     run,
+	Short:   "Display running application logs",
+	GroupID: "app-cmds",
```

Comment:
Is it a totally crazy idea to store these groupID's in variables to ensure they are the same across the cli?

Reply:
Good idea. I did that now.